LibraryApp/Book.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace LibraryApp
+{
+    public class Book
+    {
+        public int Id { get; set; }
+        public string? Title { get; set; }
+        public string? Author { get; set; }
+        public Book() { }
```

Comment:
**COMMENT** Out of curiosity do you need those two constructors? (public and non-public).

As an additional need you can read also on:
- [primary constructors](https://andrewlock.net/an-introduction-to-primary-constructors-in-csharp-12/),
- [required and init properties](https://antondevtips.com/blog/csharp-init-only-and-required-properties)

You might not need to use them, but at least worth considering.